microk8s-resources/actions/disable.metrics-server.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $SNAP/actions/common/utils.sh
+
+echo "Disabling Metrics-Server"
+
+KUBECTL="$SNAP/kubectl --kubeconfig=${SNAP_DATA}/credentials/client.config"
+# Clean up old metrics-server  
+$KUBECTL delete configmap -n kube-system metrics-server-config || true
+$KUBECTL delete deployment -n kube-system metrics-server-v0.2.1 || true 
+
+use_manifest metrics-server delete 
+# wait for pod to terminate before restarting kubelet
+echo "Waiting for pods to be terminated."
+sleep 10
+
+skip_opt_in_config "authentication-token-webhook" kubelet
+skip_opt_in_config "authorization-mode" kubelet
```

Comment:
What you are doing here is correct. Sometimes we do not do a clean disable because we do not know if other addons/workloads are using the arguments the addon enabled. I think we should leave the kubelet as is.

Reply:
I'll update this one. Thanks.